src/Compositor.cpp

```diff
@@ -560,12 +560,14 @@ void findExtensionForVector2D(wlr_surface* surface, int x, int y, void* data) {
 }
 
 CWindow* CCompositor::vectorToWindowIdeal(const Vector2D& pos) {
-    const auto PMONITOR = getMonitorFromVector(pos);
+    const auto PMONITOR     = getMonitorFromVector(pos);
+    const auto FOCUS_EXTENT = g_pConfigManager->getConfigValuePtr("general:resize_on_borders")->intValue ? g_pConfigManager->getConfigValuePtr("general:border_size")->intValue : 0;
```

Comment:
both vars should be static and then FOCUS_EXTENT should be using the static vars

Reply:
not sure if this is what you want, pls double check!